Neuro/Minigames/Completion/Solvers/RecordTemperatureSolver.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Neuro.Cursor;
+using System.Collections;
+using UnityEngine;
+
+namespace Neuro.Minigames.Completion.Solvers;
+
+[MinigameSolver(typeof(TempMinigame))]
+public sealed class RecordTemperatureSolver : MinigameSolver<TempMinigame>
+{
+    public override IEnumerator CompleteMinigame(TempMinigame minigame, NormalPlayerTask task)
+    {
+        // TODO: Figure out if the buttons are stored somewhere so we dont have to do this
```

Comment:
The buttons aren't stored anywhere so the only option is to get them using transform.Find()